README.rst

```diff
@@ -242,6 +242,7 @@ The available parsers are:
 * ``netdiff.CnmlParser``: parser for `CNML 0.1 <http://en.wiki.guifi.net/wiki/CNML>`_
 * ``netdiff.NetJsonParser``: parser for the `NetJSON NetworkGraph`_ format
 * ``netdiff.OpenvpnParser``: parser for the `OpenVPN status file <https://community.openvpn.net/openvpn/wiki/Openvpn24ManPage>`_
+* ``netdiff.WireguardParser``: parser for the `Wireguard json dump <https://github.com/WireGuard/wireguard-tools/blob/master/contrib/json/wg-json>`_
```

Comment:
Are you sure the parser is for `Wireguard json dump`? Aren't we parsing simply `wg show all dump`?

I would write:

```
parser for the Wireguard VPN (the command to use is ``wg show all dump``)
```